WISE-Community/WISE-Client
# feat(Authoring): Add copy button to each component

## Changes

Add copy button to each component. The copy button shows up when the component is hovered or expanded.

## Test

- Make sure the copy button on the component works properly
- Make sure copying multiple components works

Closes #1296
